perfkitbenchmarker/aws/aws_virtual_machine.py

```diff
@@ -143,6 +145,9 @@ def __init__(self, vm_spec):
     self.region = self.zone[:-1]
     self.image = self.image or GetImage(self.machine_type, self.region)
     self.user_name = FLAGS.aws_user_name
+    if self.machine_type in NUM_LOCAL_VOLUMES:
```

Comment:
If we support multiple local disks should we also support striping?

Seems like most local disks will be striped with mdadm